packit/utils/repo.py

```diff
@@ -20,6 +20,52 @@
 logger = logging.getLogger(__name__)
 
 
+class RepositoryCache:
+    def __init__(self, cache_path: Path, add_new=False) -> None:
```

Comment:
I'm thinking if `add_new` is good enough design here or if we want to add new things based on a filter or allowlist.

Reply:
Good point. I'll think about that more.

Reply:
Note that I would like to turn this off for service.

Reply:
yup, the weekly regeneration job could have this set to True